cogs/role_assignment.py

```diff
@@ -37,6 +39,26 @@
     "Postgraduate",
 ]
 
+ROLE_COLOURS = [
+    0x1abc9c,
+    0x11806a,
+    0x2ecc71,
+    0x1f8b4c,
+    0x3498db,
+    0x206694,
+    0x9b59b6,
+    0x71368a,
+    0xe91e63,
+    0xad1457,
+    0xf1c40f,
+    0xc27c0e,
+    0xe67e22,
+    0xa84300,
+    0xe74c3c,
+    0x992d22,
+    0x7289da
```

Comment:
You might want to consider adding a trailing comma here to produce a clearer diff if you need to add to the list